packages/react-storage/src/components/StorageBrowser/actions/handlers/__tests__/download.spec.ts

```diff
@@ -70,4 +71,15 @@ describe('downloadHandler', () => {
       message: errorMessage,
     });
   });
+
+  it('should fail if getUrl does not return a url', async () => {
+    mockGetUrl.mockResolvedValue({} as any);
```

Comment:
Prefer `@ts-expect-error` with a concise comment explaining the need for the invalid response